test/functional-api/integration/callouts/callouts.request.params.ts

```diff
@@ -145,3 +147,31 @@ export const getHubCallouts = async (
 
   return await graphqlRequestAuth(requestParams, userRole);
 };
+
+export const getChallengeCalloutByNameId = async (
+  hubId: string,
+  challengeId: string,
+  nameID: string
+) => {
+  const calloutsPerHub = await getChallengeData(hubId, challengeId);
+  const allCallouts =
+    calloutsPerHub.body.data.hub.challenge.collaboration.callouts;
+  const filteredCallout = allCallouts.filter((obj: { nameID: string }) => {
```

Comment:
you must use `find` here, but you don't need to use any sort of filtering because the callouts field
 accepts `IDs: [UUID_NAMEID!]`.
Also reusing queries like that, which fetch everything all the time, makes the tests slower, and the server load greater.
The same goes for `getOpportunityCalloutByNameId`.